src/components/pages/NavBar.tsx

```diff
@@ -0,0 +1,31 @@
+import cn from 'classnames';
+import { NavLink } from 'react-router-dom';
+import { FC } from 'react';
+
+export const Navbar: FC = () => {
+  const getLinkClass = ({ isActive }: { isActive: boolean }) =>
+    cn('navbar-item', {
+      'has-background-grey-lighter': isActive,
+    });
+
+  return (
+    <nav
+      data-cy="nav"
+      className="navbar is-fixed-top has-shadow"
+      role="navigation"
+      aria-label="main navigation"
+    >
+      <div className="container">
+        <div className="navbar-brand">
+          <NavLink className={getLinkClass} to="/">
```

Comment:
The `getLinkClass` function expects an object with a property `isActive` but you are passing it as a `className` which will not work. The `NavLink` component has an `isActive` prop which is a function that determines whether the link should be active or not. You should use this prop to determine the class for the link.